tests/scripts/run-xunit-perf.py

```diff
@@ -0,0 +1,464 @@
+
+    runArgs = [python, os.path.join(benchviewPath, 'measurement.py')] + lvMeasurementArgs + [filename]
+    log('')
+    log(" ".join(runArgs))
+
+    proc = subprocess.Popen(runArgs, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (out, err) = proc.communicate()
+    if not proc.returncode == 0:
+        print_error(out, err, 'Call to measurement.py failed.')
+
+def upload_to_benchview(python, coreclrRepo, benchviewPath, uploadToBenchview, benchviewGroup, runType, configuration, operatingSystem, etwCollection, optLevel, jitName, pgoOptimized, architecture):
+    measurementJson = os.path.join(os.getcwd(), 'measurement.json')
+    buildJson = os.path.join(coreclrRepo, 'build.json')
+    machinedataJson = os.path.join(coreclrRepo, 'machinedata.json')
+    submissionMetadataJson = os.path.join(coreclrRepo, 'submission-metadata.json')
+
+    if not os.path.isfile(measurementJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % measurementJson)
+    if not os.path.isfile(buildJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % buildJson)
+    if not os.path.isfile(machinedataJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % machinedataJson)
+    if not os.path.isfile(submissionMetadataJson):
+        raise Exception('%s does not exist. There is no data to be uploaded.' % submissionMetadataJson)
+
+    runArgs = [python,
+            os.path.join(benchviewPath, 'submission.py'),
+            measurementJson,
+            '--build',
+            buildJson,
+            '--machine-data',
+            machinedataJson,
+            '--metadata',
+            submissionMetadataJson,
+            '--group',
+            benchviewGroup,
+            '--type',
+            runType,
+            '--config-name',
+            configuration,
+            '--config',
+            'OS',
+            operatingSystem,
+            '--config',
+            'Profile',
+            etwCollection,
+            '--config',
+            'OptLevel',
+            optLevel,
+            '--config',
+            'JitName',
+            jitName,
+            '--config',
+            'PGO',
+            pgoOptimized,
+            '--architecture',
+            architecture,
+            '--machinepool',
+            'PerfSnake']
+    log('')
+    log(" ".join(runArgs))
+
+    proc = subprocess.Popen(runArgs, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (out, err) = proc.communicate()
+
+    if not proc.returncode == 0:
+        print_error(out, err, 'Createing BenchView submission data failed.')
+
+    if uploadToBenchview:
+        runArgs = [python,
+                os.path.join(benchviewPath, 'upload.py'),
+                'submission.json',
+                '--container',
+                'coreclr']
+        log(" ".join(runArgs))
+        proc = subprocess.Popen(runArgs, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Similar to above comment on `check_call`. Maybe write a helper function to handle this common case calling.